### DISCUSSION TITLE
Javascript (not Typescript) based Dev Environment

### DISCUSSION THREAD

**Discussion by attiqeurrehman** [teambit/bit#8052] [Help]
Hi, 

I recently came across `Bit.dev` and found it very impressive. However, I've only seen TypeScript-based environments so far.

Could someone point me in the right direction for using `Bit.dev` with JavaScript?

**Comment by ranm8** [comment]
You can build JS using the TypeScript environments as well. Just write JS files instead. You can also creates your own templates for JavaScript. I would recommend you to use types for shortening the feedback loop on potential issues and ease consumption experience of components.
